src/plots/cartesian/axes.js

```diff
@@ -571,34 +571,44 @@ axes.calcTicks = function calcTicks(ax) {
     if((ax._tmin < startTick) !== axrev) return [];
 
     // return the full set of tick vals
-    var vals = [];
+    var tickVals = [];
     if(ax.type === 'category' || ax.type === 'multicategory') {
         endTick = (axrev) ? Math.max(-0.5, endTick) :
             Math.min(ax._categories.length - 0.5, endTick);
     }
 
+    var isDLog = (ax.type === 'log') && !(isNumeric(ax.dtick) || ax.dtick.charAt(0) === 'L');
```

Comment:
@archmoj Nice!

Could you share a screenshot of what your `log-axis_no-minor_suffix-prefix` looked like before 27067a6c8f45dc0fbfa19dbaad0393047773601c 

Thanks!

Reply:
![before4064](https://user-images.githubusercontent.com/33888540/61662968-8acc5280-ac9d-11e9-83c6-6618c5c15be2.png)

Reply:
Awesome - thanks!!